incubator/twitter/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "kubeless-python-simple",
+  "version": "0.0.1",
+  "description": "This example demonstrates how to setup a simple Python function with Kubeless",
+  "dependencies": {
+    "bluebird": "^3.5.0",
```

Comment:
I didn't realize these modules were here. They should be dependencies of the serverless-kubeless module.